README.md

```diff
@@ -31,6 +31,44 @@ pipeline:
 - name: "rasa_composite_entities.CompositeEntityExtractor"
 ```
 
+An optional configuration variable `composite_entity_patterns` can be used to specify the filename where the composite entity patterns are defined:
+
+```yaml
+language: "en_core_web_md"
+
+pipeline:
+- name: "SpacyNLP"
+- name: "SpacyTokenizer"
+- name: "SpacyFeaturizer"
+- name: "CRFEntityExtractor"
+- name: "SklearnIntentClassifier"
+- name: "rasa_composite_entities.CompositeEntityExtractor"
+  composite_entity_patterns: "path/to/composite_entity_patterns.json"
+```
+
+`composite_entity_patterns` can be defined as the path to the parent training file:
+
+```json
+{
+    "rasa_nlu_data": {
+        "composite_entities": [],
+        "common_examples": [],
+        "regex_features" : [],
+        "lookup_tables"  : [],
+        "entity_synonyms": []
+    }
+}
+```
+
+or in as the path to a stand-alone json file:
+
+```json
+{
+    "composite_entities": []
+}
+```
+
+
 ## Usage
```

Comment:
I would prefer if the explanation about the optional path parameter would be placed into the "Usage" section. Also, you should clarify that either option works, but that giving a path to an external composite patterns file is the preferred option.